yt_dlp/YoutubeDL.py

```diff
@@ -750,6 +713,73 @@ def preload_download_archive(fn):
 
         self.archive = preload_download_archive(self.params.get('download_archive'))
 
+    @staticmethod
+    def make_logger(params):
+        screen = sys.stderr if params.get('logtostderr') else sys.stdout
+        stdout = sys.stderr if params.get('quiet') else screen
+        verbosity = (
+            Verbosity.VERBOSE if params.get('verbose')
+            else Verbosity.QUIET if params.get('quiet')
+            else Verbosity.NORMAL)
+
+        use_term_codes = not params.get('no_color', False)
+        if use_term_codes:
+            use_term_codes = None
```

Comment:
This is only there to keep compat for now. It would eventually be replaced by a commandline flag (`--term-codes 'stdout:false`) or parameter (`{"term_codes": {"stdout": True}}`) allowing to force a specific behavior.
Since the base framework is there this could be postponed until after the merge.